code/frameworks/angular/src/client/angular-beta/ComputesTemplateFromComponent.ts

```diff
@@ -73,19 +73,19 @@ const createAngularInputProperty = ({
   value: any;
   argType?: ArgTypes[string];
 }) => {
-  const { name: type = null } = (typeof argType?.type === 'object' && argType?.type) || {};
-  let templateValue = type === 'enum' && value;
-
-  const actualType = type === 'enum' && typeof value;
-  const requiresBrackets = ['object', 'any', 'boolean', 'enum', 'number'].includes(actualType);
-
-  if (typeof value === 'object') {
-    templateValue = propertyName;
+  let templateValue;
+  switch (typeof value) {
+    case 'string':
+      templateValue = `'${value}'`;
+      break;
+    case 'object':
+      templateValue = JSON.stringify(value).replace(/"/g, "'");
```

Comment:
I don't think this is a safe operation.

Let's consider the following object:

```jsx
const obj = {"hello":"This is a nice 'sentence'"}
```

JSON.stringify will convert it to:

```jsx
const value = `{"hello":"This is a nice 'sentence'"}`
```

The replace operation would output:
```jsx
const replaced = "{'hello':'This is a nice 'sentence''}"
```